rfcs/text/0001-rfc-process.md

```diff
@@ -70,26 +88,66 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs. Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in the DCP Zenhub board.
+
+To inform the DCP Community about the development of an RFC, with the intention of forming collaborations between people thinking about similar problems, it is recommended that a draft of an RFC be first published once the Problem Statement and Authors have been established. One possible process that is recommended is as follows:
+
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
+   5. Tag the PR with the "rfc-draft" label.
+
+#### **Non-Authors**:
+
+When an RFC _Draft_ PR is created and furthermore labeled with "rfc-draft" commentary is not allowed since the RFC is under active development. This time period is specifically an opportunity for collaboration on authorship so if an RFC sparks interest and you would like to collaborate on it (especially if you have been thinking about the problem statement yourself), reach out the Author(s) directly and offline. Specific commentary whether agreement or disagreement on the RFC is discouraged until the Author(s) is/are ready for the PR to be proposed/reviewed.
+
+**ADDITION ENDS**
+
 ### Proposing RFCs
 
 #### **Authors**:
 
-*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+**UPDATE BEGINS**
 
-  - Fork the HumanCellAtlas dcp-community repository
-  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
-  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
+~~*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*~~
+
+Once the RFC has been filled out to the Author's satisfaction, an RFC may be _proposed_ for comment.
```

Comment:
Editorial: There's a redundant _Fill in the RFC with attention to detail_ on Line 131. 

I can imagine the author - "Make up your mind! - My "satisfaction" or "attention to detail". Which? Which! I hate details!

Reply:
Just deleted this sentence :)